Code/Tools/SceneAPI/SceneUI/ManifestMetaInfoHandler.cpp

```diff
@@ -33,9 +40,10 @@ namespace AZ
 
             void ManifestMetaInfoHandler::GetIconPath(AZStd::string& iconPath, const DataTypes::IManifestObject& target)
             {
+                // Icons for classes that don't have edit serialize contexts
                 if (target.RTTI_IsTypeOf(DataTypes::IMeshGroup::TYPEINFO_Uuid()))
                 {
-                    iconPath = ":/SceneUI/Manifest/MeshGroupIcon.png";
```

Comment:
It looks like this is the only place that used this `/SceneUI/Manifest/MeshGroupIcon.png` is used, so can we delete it?

Reply:
done, deleted